frontend/packages/dev-console/src/components/topology/data-transforms/ModelContext.ts

```diff
@@ -162,6 +170,22 @@ export class ExtensibleModel {
     return this.modelP;
   }
 
+  public set loaded(loaded: boolean) {
+    this.loadedP = loaded;
+  }
+
+  public get loaded(): boolean {
+    return this.loadedP;
+  }
+
+  public set loadError(loadError: string) {
+    this.loadErrorP = loadError;
+  }
+
+  public get loadeError(): string {
+    return this.loadErrorP;
+  }
```

Comment:
Why bother with getters and setters instead of simply making the properties themselves public?